### PR TITLE
add yaml_cpp_struct

### PR DESCRIPTION
**Pull request by fantasy-peak**
https://github.com/xmake-io/xmake-repo/issues/2013

### CODE REVIEW

File: packages/v/visit_struct/xmake.lua [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package("visit_struct")
+    set_homepage("https://github.com/cbeck88/visit_struct")
```
Comment (waruqi) [inline packages/v/visit_struct/xmake.lua:2]: set_kind("library", {headeronly = true})

File: packages/y/yaml_cpp_struct/xmake.lua [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package("yaml_cpp_struct")
```
Comment (waruqi) [inline packages/y/yaml_cpp_struct/xmake.lua:1]: and here

Comment (fantasy-peak) [reply packages/y/yaml_cpp_struct/xmake.lua:1]: done

File: packages/v/visit_struct/xmake.lua (hunk 1 above)
Comment (fantasy-peak) [reply packages/v/visit_struct/xmake.lua:3]: done

**Pull request merged**